include/rocksdb/comparator.h

```diff
@@ -78,6 +78,12 @@ class Comparator {
   // The major use case is to determine if DataBlockHashIndex is compatible
   // with the customized comparator.
   virtual bool CanKeysWithDifferentByteContentsBeEqual() const { return true; }
+
+  virtual size_t TimestampSize() const { return 0; }
```

Comment:
Do we have to make it virtual here? Can we just use a variable?

Reply:
Yes we can.